### PR TITLE
UHM-6643: Lab OWA Assessments and Tweaks

### PR DESCRIPTION
**Pull request by mogoodrich**
This PR basically does two things:

* Now that Cosmin has added the "lab categories" filter, I removed the confusingly named "lab results to display concept set" GP I had created and use the lab categories filter to determine what is displayed in the Lab Results list.  It seemed needlessly confusing to have two.  It's still a bit clunky because it now created two constants, the "lab categories" (used for the category filters) and the "lab categories flattened" (used for top-level filtering) but this seemed the most bang for the buck
* I created a new "labowrkflowowa.labCategoriesConceptSet" to the define what concept is used to define the lab categories, so that this set can diverge from the sert of orderables if needed.

### CODE REVIEW

**Comment by mogoodrich** [comment]
Oops, looks like I have some tests to fix!

File: app/js/utils/globalProperty.js [hunk 1]
```diff
@@ -21,12 +22,12 @@ export const APP_GLOBAL_PROPERTIES = {
   labResultsTestLocationQuestion: "labworkflowowa.locationOfLaboratory",
   labResultsEstimatedCollectionDateQuestion: "labworkflowowa.estimatedCollectionDateQuestion",
   labResultsEstimatedCollectionDateAnswer: "labworkflowowa.estimatedCollectionDateAnswer",
-  labResultsToDisplayConceptSet: "labworkflowowa.labResultsToDisplayConceptSet",
   dateAndTimeFormat: "labworkflowowa.dateAndTimeFormat",
   enableLabelPrinting: "labworkflowowa.enableLabelPrinting",
   labelPrintingEndpoint: "labworkflowowa.labelPrintingEndpoint",
   ordersBatchSize: "labworkflowowa.ordersBatchSize",
   orderLabTestLink: "labworkflowowa.orderLabTestLink",
+  labCategoriesConceptSet: "labworkflowowa.labCategoriesConceptSet",
```
Comment (cioan) [inline app/js/utils/globalProperty.js:30]: The new labCategoriesConceptSet GP you added points to the same UUID as the labOrderablesConceptSet GP below. I think that you could safely remove the labOrderablesConceptSet GP and use the new one you just introduced.

**Review by cioan — APPROVED** [review]
Looks good @mogoodrich ! Per my comment below, i think you could remove the labOrderablesConceptSet GP I introduced just before your PR.

File: app/js/utils/globalProperty.js (hunk 1 above)
Comment (mseaton) [reply app/js/utils/globalProperty.js:30]: Yeah, where is the labOrderablesConceptSet used in the labworkflow owa @mogoodrich .  Do we still need this?

**Review by mseaton — APPROVED** [review]

File: app/js/utils/globalProperty.js (hunk 1 above)
Comment (cioan) [reply app/js/utils/globalProperty.js:30]: @mseaton , a few places in code have to be updated if we get rid of labOrderablesConceptSet. Here is the beginning of that refactoring:
https://github.com/openmrs/openmrs-owa-labworkflow/blob/master/app/js/actions/labOrderablesAction.js#L34

Comment (mogoodrich) [reply app/js/utils/globalProperty.js:30]: Yep, good point guys... originally I had replaced the lab orderables GP with the new GP, but then released the orderables was being used elsewhere, namely when generating the test types categories on the list of lab orders page, which I was thinking maybe *should* be based on the orderables?



Anyway, my plan was to ticket that in a separate ticket, and/or make sure we handle it when/if we change the way lab orderables work?

**Comment by mogoodrich** [comment]
Thanks @mseaton  and @cioan , merging this in... (I am still going to look into removing the labOrderables GP and will comment on that in a follow-on commit and/or ticket...)

**Pull request merged**